src/components/grader/submissions/Revision.js

```diff
@@ -1,20 +1,35 @@
 import React, { useEffect, useState } from "react";
-import { Accordion, AccordionItem } from "carbon-components-react";
+import { Accordion, Button } from "carbon-components-react";
 import apiClient from "../../../api-client";
 import SingleSubmission from "./SingleSubmission"
+import { Reset32 } from '@carbon/icons-react';
 import './Revision.scss'
 
 function Revision(props) {
 
     const [fileNameList, setFileNameList] = useState();
     const [visible, setVisible] = useState(false);
     const [diff, setDiff] = useState();
+    const [update, setUpdate] = useState(0)
+    const [regrade, setRegrade] = useState(false)
 
     useEffect(() => {
         getSubmissionFilesList()
         getDiffResults();
-        // eslint-disable-next-line 
-    }, [])
+    }, [update])
+
+
+    useEffect(() => {
+        if (fileNameList)
+            setVisible(true)
```

Comment:
I guess it's not the best thing to use `if` without curly braces. It impacts readability and makes code prone to bugs